src/libs/actions/Session/index.js

```diff
@@ -97,7 +107,7 @@ function signOutAndRedirectToSignIn() {
         Linking.getInitialURL().then((url) => {
             const reportID = ReportUtils.getReportIDFromLink(url);
             if (reportID) {
-                Report.setLastOpenedPublicRoom(reportID);
+                Onyx.set(ONYXKEYS.LAST_OPENED_PUBLIC_ROOM_ID, reportID);
```

Comment:
AFAIK, it is recommended to use `Onyx.merge`.

Reply:
ah, just copied and pasted the code from the Report.setLastOpenedPublicRoom :)

Reply:
+1 to merge

Reply:
I guess it's out of the scope of this ticket to change that